src/test/java/io/jenkins/plugins/junit/storage/TestResultStorageJunitTest.java

```diff
@@ -476,7 +476,7 @@ public List<HistoryTestResultSummary> getHistorySummary(int offset) {
                                     Job<?, ?> theJob = Jenkins.get().getItemByFullName(getJobName(), Job.class);
                                     if (theJob != null) {
                                         Run<?, ?> run = theJob.getBuildByNumber(buildNumber);
-                                        historyTestResultSummaries.add(new HistoryTestResultSummary(run, null, duration, failed, skipped, passed));
```

Comment:
this shouldn't have had null passed as it made the test run but if any one actually tried to access the history view they got null pointers